README.md

```diff
@@ -1,40 +1,69 @@
-Project Name
-Menanya's Portfolio
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Menanya Morris' Portfolio
+
+> Mobile Skeleton of a web portfolio application 
+
+
+## Built With
+
+- HTML and CSS
+- Git and Github
+
+
+## Live Demo (if available)
+
+Not Available
+
+
+## Getting Started
 
-Built With
-HTML and CSS
-Frameworks
-Technologies used
-Live Demo (if available)
-Live Demo Link
 
-Getting Started
-All informations related to Morris Menanya is included has been add and uploaded to this site, carefully go through differnt sections to learn about his career growth and personal life. you can also leave a message and request for a collaboration or recuitement from the homepage
 
 
 To get a local copy up and running follow these simple example steps.
 
-Prerequisites
-Setup
-Install
-Usage
-Run tests
-Deployment
-Authors
-👤 Author1
+### Prerequisites
+  None
+
+### Setup
+Download and view file
+
+### Install
+
+### Usage
```

Comment:
- [ ] Please let's fill in the content of these headers. This will make our README file descriptive.